CorsixTH/Lua/app.lua

```diff
@@ -29,7 +29,7 @@ local runDebugger = corsixth.require("run_debugger")
 -- and add compatibility code in afterLoad functions
 -- Recommended: Also replace/Update the summary comment
 
-local SAVEGAME_VERSION = 176 -- Palette fix
+local SAVEGAME_VERSION = 177 -- Palette fix
```

Comment:
```suggestion
local SAVEGAME_VERSION = 177 -- Fix requeue action regression
```
?

Reply:
Yes, string should change, the point of the comment is to detect conflicts.